application/comit_node/src/http_api/impl_serialize_http.rs

```diff
@@ -0,0 +1,30 @@
+macro_rules! _count {
+    () => (0usize);
+    ($x:tt $($xs:tt)*) => (1usize + _count!($($xs)*));
+}
```

Comment:
❗️ this looks like you had some fun. Explain this to me tomorrow :)

Reply:
I'd love to take credit, but I took it from [here](https://stackoverflow.com/a/34324856).